client/my-sites/site-settings/composing/index.jsx

```diff
@@ -47,8 +48,13 @@ const Composing = ( {
 
 			{
 				siteIsJetpack && jetpackSettingsUISupported && (
+					<JetpackMarkdown />
+				)
+			}
+
+			{
+				siteIsJetpack && jetpackSettingsUISupported && (
```

Comment:
Why introduce the same check in the same place? Let's just use the one that's already there and place `<JetpackMarkdown />` right there.